opentech/apply/review/models.py

```diff
@@ -135,6 +136,15 @@ class Meta:
     def outcome(self):
         return self.get_recommendation_display()
 
+    @property
+    def role_review_icon(self):
```

Comment:
This isn't used anywhere i can see.

if you are planning on using it code would be:
```
try:
    role = self.submission.assgned.with_role().get(reviewer=self.author)
except AssignedReivewer.DoesNotExist:
    return None

return role.icon
```

I think i would like to look at associating the review author object with the assigned reviewer table, but that is well outside the scope of this ticket.

Reply:
Yes, I needed it earlier based on an earlier iteration of the detail template review information. It was cruft. Thanks for catching.